js/ui/text_box/ui.text_editor.mask.js

```diff
@@ -364,7 +364,7 @@ var TextEditorMask = TextEditorBase.inherit({
             this._caret({ start: caret.start, end: caret.start });
 
             this._maskKeyHandler(e, function() {
-                this._handleKey(char.charCodeAt());
+                this._handleKey(char && char.charCodeAt());
```

Comment:
Use `if(!caret.end) return;` in line 356 instead of this check. It is better because caret will set to the start of the empty input when clear button clicked on blured editor

Reply:
@mrjohn0011 

Thank you. Done.